### ISSUE TITLE
Deletes project files without finishing work!

### ISSUE THREAD

**Issue by Shinkir0** [n00mkrad/cupscale#65]
I had insufficient space on the drive where I saved the video file, so the program gave the message that there is not enough space on disk, and after I closed the window with the message the program deleted all the temporary files and never finished.

**Comment by aodgizmo** [comment]
Yes this is the worst. Waited 15+ hours for it to finish doing it task, than it had a problem moving the file out. It than proceeded to delete all the work it took 15 hrs to complete and now I have to start over. This needs a popup or something asking "Do you wish to delete all the temp files?". 

Editing a video that takes all day only for it to be deleted because it couldn't move the completed file sure does suck...
